src/main/java/org/kohsuke/github/GHTeam.java

```diff
@@ -126,6 +126,13 @@ public void add(GHRepository r) throws IOException {
     public void remove(GHRepository r) throws IOException {
         org.root.retrieve().method("DELETE").to(api("/repos/" + r.getOwnerName() + '/' + r.getName()), null);
     }
+    
+    /**
+     * Deletes this team.
+     */
+    public void delete() throws IOException {
+        org.root.retrieve().method("DELETE").to(api(""));
```

Comment:
According to [GitHub API](https://developer.github.com/v3/orgs/teams/#delete-team), the url is `/teams/:id`, which is what `api` method creates